.golangci.reference.yml

```diff
@@ -107,6 +107,15 @@ output:
 
 # All available settings of specific linters.
 linters-settings:
+  asasalint:
+    # Add FuncName to exclude for some bad case report.
+    exclude:
+      - Append
+      - Log
+    # ignore found case in *_test.go file
+    ignore-in-test: false
+    # if true, will disable the default exclude see https://github.com/alingse/asasalint/blob/main/asasalint.go#L15 like: Printf,Println,Errorf,Debugf ...
+    no-default-exclude: false
```

Comment:
the default values must be defined inside the comments.
take a look at the other linters